packages/shared/src/graphql/sources.ts

```diff
@@ -42,7 +42,7 @@ export interface Squad extends Source {
   permalink: string;
   public: boolean;
   type: SourceType.Squad;
-  owners?: string[];
+  admins?: string[];
```

Comment:
I think this is not used, `owners` and now `admins` does not exist as column or in `query Source` on API.

Reply:
Ah good point removed it now.